platforms/hyperledger-aries/chart/values.yaml

```diff
@@ -0,0 +1,164 @@
+---
+
+app:
+  config:
+    genesisFile: 'ledgers.yaml'
+    ## non-empty value is REQUIRED for genesisURL
+    genesisURL: ''
+    mountPath: '/home/indy'
+    networkName: 'NETWORK_NAME'
+  debugEnable: false
+  env: 'dev'
+  group: 'vc-authn-oidc'
+  name: 'vc-authn-oidc'
+  role: 'agent'
+
+controller:
+  role: 'controller'
+  webhookBaseURL: ''
+
+deploy:
+  image:
+    name: 'ghcr.io/hyperledger/aries-cloudagent-python'
+    ## Modify aca-py agent version: 
+    tag: 'py3.9-0.9.0'
+  name:
+    prefix: 'aries-agent'
+    suffix: ''
+
+ingress:
```

Comment:
Bevel calls ingress and proxy, so can this be moved to key called `global.proxy`